Sources/HaCWebsiteLib/ViewModels/LandingPage.swift

```diff
@@ -1,5 +1,7 @@
 import HaCTML
 
+// swiftlint:disable line_length
```

Comment:
Seems like we might have to disable this for most ViewModels 🤷‍♀️

Reply:
Yeah I think so

Alternatively we can try to keep our line lengths shorter ;)